docs/assessment/index.md

```diff
@@ -105,16 +153,56 @@ An assessment is broken down in to a list of zones, like this:
 }
 ```
 
-- Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+| Zone Property   | Type    | Description                                                                                                                                           |
+| --------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `title`         | string  | The title of the zone. (Optional; default: none)                                                                                                      |
+| `questions`     | array   | The list of slots for questions and question alternatives within the zone. (Optional; default: none)                                                  |
+| `numberChoose`  | integer | Number of questions to select for each student from this zone. (Optional; default: select all)                                                        |
+| `maxPoints`     | number  | Limit on the number of points that can be earned from this zone. (Optional; default: sum of question max points)                                      |
+| `bestQuestions` | integer | Only this many questions in the zone will count towards the total points (highest-point questions will count). (Optional; default: use all questions) |
 
-- Within each zone the question order is randomized based on [the assessment type and the `shuffleQuestions` option](#changing-question-order-randomization).
+Zone specification details are in the [format specification for `infoAssessment.json`](../schemas/infoAssessment.md)
 
-- An assessment question can be specified by either a single `id` or by a list of alternatives, in which case one or more of these alternatives is chosen at random. Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
+- Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+
+- Within each zone the question order is randomized based on [the assessment type and the `shuffleQuestions` option](#question-order-randomization).
 
 - If a zone has `maxPoints`, then, of the points that are awarded for answering questions in this zone, at most `maxPoints` will count toward the total points.
 
 - If a zone has `bestQuestions`, then, of the questions in this zone, only `bestQuestions` with the highest number of awarded points will count toward the total points.
 
+## Slots for questions and question alternatives
+
+Each zone has a list of _slots_ given by the `questions` array. Each slot contains either a single question `id`:
```

Comment:
We're introducing a new term here. Maybe this is the right call given that "alternative groups" or "zone questions" (which we use internally) aren't super friendly names; I just want to make sure we're thinking this through.

Reply:
The "slot" terminology is one that I often use when giving talks about PL. But I think here it isn't really adding much, and I don't like using terminology in the docs which isn't reflected in the code. I removed "slots" in cdd94bc86d95f807483661a098282fe48aace1fc in favor of talking about "entries in the questions list".